app/assets/stylesheets/components/_important-metadata.scss

```diff
@@ -30,7 +29,12 @@
 .app-c-important-metadata__definition {
   font-weight: bold;
 
-  a:focus {
-    @include govuk-focused-text;
+  .govuk-link {
```

Comment:
I wonder if it's clearer to add another class (for example, `gem-c-link--inverse`), rather than overriding `govuk-link`? Adding another class to extend seems to be the [technique that's recommended](https://design-system.service.gov.uk/get-started/extending-and-modifying-components/#avoid-overwriting-gov-uk-design-system-code).